src/main/java/net/snowflake/client/core/PrivateLinkDetector.java

```diff
@@ -6,6 +6,9 @@ public class PrivateLinkDetector {
    * We can only tell if private link is enabled for certain hosts when the hostname contains the
    * word 'privatelink' but we don't have a good way of telling if a private link connection is
    * expected for internal stages for example.
+   *
+   * @param host host
+   * @return true if host string contains the word 'privatelink'
```

Comment:
it's implementation detail - let's change to `true if host is considered as privatelink environment`